app/models/mixins/compliance_mixin.rb

```diff
@@ -9,7 +9,9 @@ module ComplianceMixin
              :inverse_of => :resource,
              :class_name => "Compliance"
 
-    supports :check_compliance
+    supports :check_compliance do
+      unsupported_reason_add(:check_compliance, "No compliance policies assigned") unless has_compliance_policies?
```

Comment:
Do we have a way to i18n the text, or is that done later?

Reply:
https://github.com/kbrock/manageiq/blob/compliance_policies/app/models/mixins/compliance_mixin.rb#L47-L49

Reply:
There hasn't been i18n. I can add _ here